src/amplitude-wrapper.js

```diff
@@ -156,10 +156,16 @@ const LOG_PREFIX = '[Amplitude / GTM]';
           .setPrice(args.price)
           .setRevenueType(args.revenueType || '')
           .setEventProperties(args.eventProperties || {})
-          .setRevenue(args.revenue || (args.price * (args.quantity || 1)))
-          .setCurrency(args.currency || '')
-          .setReceipt(args.receipt || '')
-          .setReceiptSig(args.receiptSig || '');
+          .setRevenue(args.revenue || (args.price * (args.quantity || 1)));
+          if (args.currency) {
+            revenue.setCurrency(args.currency);
+          }
+          if (args.receipt) {
+            revenue.setReceipt(args.receipt);
+          }
+          if (args.receiptSig) {
+            revenue.setReceiptSig(args.receiptSig);
+          }
```

Comment:
Can we do the same on revenue type and event properties? They are optional too https://github.com/amplitude/Amplitude-TypeScript/blob/e736a65e35f0462a6b6080e66a27ffaec9a512b5/packages/analytics-core/src/revenue.ts#L16-L25 
![image](https://github.com/user-attachments/assets/2e74325c-34fc-41f5-901f-95956dd567ae)